cmd/oci-image-tool/cas_get.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2016 The Linux Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/opencontainers/image-spec/image/cas/layout"
+	"github.com/spf13/cobra"
+)
+
+type casGetCmd struct {
+	stdout io.Writer
+	stderr *log.Logger
+	path   string
+	digest string
+}
+
+func newCASGetCmd(stdout io.Writer, stderr *log.Logger) *cobra.Command {
+	state := &casGetCmd{
+		stdout: stdout,
+		stderr: stderr,
+	}
+
+	return &cobra.Command{
+		Use:   "get PATH DIGEST",
+		Short: "Retrieve a blob from the store",
+		Run:   state.Run,
+	}
+}
+
+func (state *casGetCmd) Run(cmd *cobra.Command, args []string) {
```

Comment:
On Mon, Jun 20, 2016 at 02:19:47PM -0700, Sergiusz Urbaniak wrote:
> > +func (state *casGetCmd) Run(cmd *cobra.Command, args []string) {
> 
> Haha, sorry, it's 11pm here in old Europe, my eyes are starting to
> blur ;-) In this case I'd recommend calling it `getCmd`, or even
> `cgc` which is quite idiomatic.

getCmd would cause collisions between the current casGetCmd and
refsGetCmd.  Do you really prefer types ‘cgc’ and ‘rgc’ to casGetCmt
and refsGetCmd?  I can reroll (it's not my project ;), but the short
versions are a bit dense for my taste ;).